lib/vue/src/components/webcam/CWebcamButton.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <b-button
+    variant="light"
+    class="d-flex align-items-center"
```

Comment:
This class should be provided by the parent component, since different views can have differently styled buttons.